docs/api.md

```diff
@@ -453,6 +457,40 @@ Ractive({
 })
 ```
 
+*Example 2*: Attaching, detaching, firing events, creating two-way bindings within the template
+
+<div data-playground="N4IgFiBcoE5SBTAJgcwSAvgGhAZ3gLICeAwgPYC2ADmQHYK0AuABALzMBKAhgMaMCWANwQA6BAA9GDJAApgAHVrNmU6gBsuUyMwAGi5coA8AIwCujRnWZ0AtDzX8eAa1byQ9x07cA+XFyLMYAhqamQAhMwyMNIIMMw8ZKZM2sDACUmMGBgAlJEUCLh+aCnA+YVcaFnZhgD0ZhZ03vq6WM100bRIsdoAZkl8-HQy2QpKzPIWYPy4IlxIsgDk6UwL2QDczdjNSJpcvf0CQyPNE9GMpjBKo8oTE8uM2gAMJ5mbijmKitwDwnLNjFwYGgHswFsYyEgiAtWmMdgCUs1lBRcChtAsABLBUJYZgAdTIMDUSDCC02MOUqioGi0uheJnMliUtg8zlcIAoZGEPg5wmY-EYtXqjO84zGE3pDSZtDsDlZbmBACFeF4QN5gcxjMrBQzGp8xQKkEJmLhGEQ1Ag2eCYF0YNoAMxUcTGsgOJDMaJINaBBD8FBgEEAVkejp8nxYBkMAGJmByMjR+EwbHQENZpSyXG4giEyG4YwUiha3KlkSgsrmak0xkYaobBJWbvrDLXjabzZaCTb7Y7na7mEQsWQAO5eoK+-3aIMh1UvIzR2NMeOJxiDsipmWeNlZ0K5soFtmpGMo5gAalBJ7Pp5LzDLIGYFcRzFqtfrBgfhjAdu8REScV3FRTQTRLUH4vkYCZUOYzCCFwaimIWIDFiiN53i+OjkqmHSdswfS0AMRzXAYNQ1MwJDRJoKb0IOfK0CatC8AgD4UEQACSNEArhKbsMQ5DUMmTDDBstC3JM0yzBYvBgCQUxEjITGsSaXAcTiwAqICwJovOjCLowSb0AsHxCYwEyMFMMxDKMwkTDyCD7Lhhy0MMFlGc5xmmSIXQAjwknSbIclsYpPAIOsYaWSJMyaJ53n8DJfkKUpzAqQCQIICCCyadpNjLmQ+nZCFzlbIZwmKsqtl4Q5xyFS5YXuSlElSdFvksf5HHBZVwkmaJEV1T5slNXFgXKapyWpelZAJjpyY5XlEwFaFCxbtlpX2TIfDiBVoVWYkTAAApjUwbDxIw4giAkvH0EwJ2UDQ52MCIdH5NNzkwbEjAyJmA7YTAlDjLel5bYwu3jeeuaDvyYB5uUxQ-eeq2XWdDC3cCMhpfm-6rLlbWvJVBnKBg7zZJgQA"></div>
+
+```js
+MyComponent = Ractive.extend({
+    template: ...
+})
+
+Ractive({
+    ...
+    template: `
+        <button on-click="move">move it</button>
+	    <# mountpoint-one on-click="hello" message="{{msg}}" />
+        <# mountpoint-two on-click="hello" message="{{ msg + ' + ' + msg }}" />
+        <input value="{{msg}}" />
+    `,
+    onrender: function(){
+        // Create new instnace
+        myInstance = MyComponent();
+		this.attachChild(myInstance, { target: 'mountpoint-one'})
+		this.on({
```

Comment:
Is it me, Github, or something's off with the indent?

Reply:
Even though it is displayed well in my screen: 

![image](https://user-images.githubusercontent.com/6639874/30312132-de99c8e8-97a0-11e7-92fd-202e84c33f47.png)

yes, it looks weird in the code if user's (or Github's) tab size is different from 4 spaces. 

It is because of the mixed TABs and spaces: I create the examples in the playground (which uses TAB for indentation), and edit its visual representation in my editor, which uses 4 spaces instead of TABs. 

Switching between TABs and spaces is really a pain. Which style should we follow? 4 spaces or TABs?